### ISSUE TITLE
Additional iNaturalist tab on leave's description 

### ISSUE THREAD

**Issue by oolonek** [OneZoom/OZtree#864]
Related to https://github.com/OneZoom/OZtree/issues/863

This would also be great to acess :

- The taxon's iNaturalist page e.g. for https://www.onezoom.org/life/@Aloe_ferox=608115 this would be https://www.inaturalist.org/taxa/124412-Aloe-ferox
- Also a link to all `observations` of this given taxa could be a thing. Here https://www.inaturalist.org/observations?taxon_id=124412

**Comment by hyanwong** [comment]
Yes, iNat would be nice. We would need to add the iNat ID to the mappings though, which might not be obvious (I assume we might get it from wikidata).

Are you interested in adding this @oolonek ? The identifier mapping code could probably do with some reworking. This should be a lot easier now that @davidebbo has rewritten the tree production code (and added unit tests etc).

**Comment by davidebbo** [comment]
> I assume we might get it from wikidata

Yes, it is there as [P3151](https://www.wikidata.org/wiki/Property:P3151). So we'd need to add a new column to the ordered_leaves table.

**Comment by hyanwong** [comment]
This will probably mainly be done in the https://github.com/OneZoom/tree-build/ repo, I think?

**Comment by davidebbo** [comment]
Yes, I think the work is:
- Add inat column to leaves table
- Update tree-build to get the mapping and produce a .csv.mySQL that has it
- Add logic to OZtree to consume this column and add a UI tab.
